### PR TITLE
[ja] translate behaviors.rst

### PR DESCRIPTION
**Pull request by natsukinoue**

### CODE REVIEW

**Comment by chinpei215** [comment]
@natsukinoue Thank you for your contribution. Could you please translate comments in examples?

File: ja/orm/behaviors.rst [hunk 1]
```diff
@@ -19,3 +22,250 @@ Behaviors
...
+
+テーブルからビヘイビアを削除するには、 ``removeBehavior()`` メソッドを呼び出します。 ::
+
+    // Remove the loaded behavior
+    $this->removeBehavior('Sluggable');
+
+ロードされたビヘイビアへのアクセス
+===================================
+
+ビヘイビアをテーブルインスタンスに追加したら、読み込まれたビヘイビアをイントロスペクトしたり、 ``BehaviorRegistry`` を使用して特定のビヘイビアにアクセスしたりできます。 ::
```
Comment (chinpei215) [inline ja/orm/behaviors.rst]: > イントロスペクト

`確認` would be a better wording.

File: ja/orm/behaviors.rst [hunk 2]
```diff
@@ -19,3 +22,250 @@ Behaviors
...
+        }
+
+    }
+
+上記のコードは、ビヘイビアの興味深い機能をいくつか示しています。
+
+- ビヘイビアでは、:ref:`table-callbacks` に従うメソッドを定義することで、コールバックメソッドを定義できる。
+- ビヘイビアでは、デフォルトのコンフィグレーションプロパティを定義できる。ビヘイビアがテーブルに追加されている場合、このプロパティはオーバーライドとマージされる。
+
+継続して保存されないようにするには、コールバック内のイベント伝播を停止するだけです。 ::
```
Comment (chinpei215) [inline ja/orm/behaviors.rst]: > To prevent the saving from continuing

保存が続行しないようにするには

File: ja/orm/behaviors.rst (hunk 1 above)
Comment (natsukinoue) [inline ja/orm/behaviors.rst]: Thank you for the comment. I searched some Japanese documents about "introspect" and found the words "インストロペクト" or "インストロペクション" in most of them. (e.g.  https://docs.oracle.com/javase/jp/6/api/java/beans/Introspector.html) Literally introspection is  "内省" in Japanese but I don't believe 内省 is good word for this context. So I'd like to leave this part as it is. What do you think?

File: ja/orm/behaviors.rst [hunk 3]
```diff
@@ -19,3 +22,249 @@ Behaviors
...
+            $this->addBehavior('Sluggable');
+        }
+    }
+
+この新しいビヘイビアは、今は何もしません。 次に、ミックスインメソッドとイベントリスナーを追加して、エンティティを保存するときにフィールドを自動的にslugさせることができるようにします。
+
+ミックスインメソッドの定義
+--------------------------
+
+ビヘイビアに定義されたパブリックメソッドは、それが追加されたテーブルオブジェクトに 'mixin'メソッドとして追加されます。 同じメソッドを提供する2つのビヘイビアを追加すると、例外が発生します。 ビヘイビアがテーブルクラスと同じメソッドを提供する場合、ビヘイビアメソッドはテーブルから呼び出すことはできません。 ビヘイビアミックスインメソッドは、テーブルに提供されるものとまったく同じ引数を受け取ります。 たとえば、SluggableBehaviorが次のメソッドを定義しているとします。 ::
```
Comment (okinaka) [inline ja/orm/behaviors.rst]: > 'mixin'

「ミックスイン」

File: ja/orm/behaviors.rst [hunk 4]
```diff
@@ -19,3 +22,249 @@ Behaviors
...
+                // 必要であれば値を変換
+                return $value . '123';
+            }
+        ];
+    }
+
+``TranslateBehavior`` には、参照される可能性のあるこのインターフェースの重要な実装があります。
+
+.. versionadded:: 3.3.0
+    The ability for behaviors to participate in marshalling was added in 3.3.0
```
Comment (okinaka) [inline ja/orm/behaviors.rst]: This line is not translated.

File: ja/orm/behaviors.rst (hunk 1 above)
Comment (chinpei215) [inline ja/orm/behaviors.rst]: @natsukinoue I am sure that 99% of Japanese people cannot understand what `イントロスペクト` means. So you searched information about the word, didn't you? If you would like to keep "introspect", you can do like [this example](http://lfsbookja.osdn.jp/systemd.ja/chapter06/systemd.html#networkctl).

**Comment by okinaka** [comment]
@natsukinoue Thanks!

**Pull request merged**